wiki/Article_Style_Guide/en.md

```diff
@@ -1,627 +1,468 @@
+        - You can also mention contributors, issues, or pull request numbers.
+4. Push to GitHub.
 ```
 
-There are two kinds of lists:
-
-- bulleted
-  - used when the order of each item does not matter (e.g. describing an object)
-- numbered
-  - used when the order of each item does matter (e.g. tutorial instructions)
+### Code
 
-### Sub-Lists
+The markup for code is a grave mark (`` ` ``).
+To put a grave mark in code, surround it in double grave marks, padded with 1 space.
 
-Sub-lists are lists that are indented underneath a listed item.
-
-Please limit to a level of four (4) sub-lists at a time (if you need more, you should reconsider about how you are structuring the list).
+**Example:**
 
 ```
-- item A
-  - sub-item A.1
-    - sub-sub-item A.1.i
-      - sub-sub-sub-item A.1.i.a
-      - sub-sub-sub-item A.1.i.b
-    - sub-sub-item A.1.ii
-      - sub-sub-sub-item A.1.ii.a
-    - sub-sub-item A.1.iii
-  - sub-item A.2
-    - sub-sub-item A.2.i
-  - sub-item A.3
-- item B
+Press `Z` and/or `X` to tap.
+Press `` ` `` to quick retry a map.
 ```
 
-## Code
-
 For the following scenarios, use code.
 
-### Keyboard Keys
+#### Keyboard Keys
 
 When doing this, use capital letters for single characters and [camel case](https://en.wikipedia.org/wiki/Camel_case) for modifiers.
 
-For example:
+**Examples:**
 
 - `` `.` ``
 - `` `Alt` ``
 - `` `Ctrl` + `Shift` + `A` ``
 
-### Button or Menu Text
+#### Button or Menu Text
 
 When copying the text from a menu or button, copy the letter casing as-is.
-For example:
+
+**Examples:**
 
 - `` Click `File` then `Save As...` ``
 - `` `1. Let's do it.` ``
 - `` `4. Noooo! I didn't mean to!` ``
 
-### Folder Names
+#### Folder Names
 
 When copying the name of a folder, copy the letter casing as-is.
-For example:
+
+**Examples:**
 
 - `` `Exports` ``
 - `` `Downloads` ``
 
-### Directory Names
+#### Directory Names
 
 When copying the path of a directory, copy the letter casing as-is.
-For example:
+
+**Examples:**
 
 - `` `Downloads/` ``
 - `` `Skins/User/` ``
 
-### Keywords or Commands
+#### Keywords or Commands
 
 When copying a keyword or command, copy the letter casing as-is or as you would normally type it as (prefer lowercase if applicable).
-For example:
+
+**Examples:**
 
 - `` `ComboBurstStyle` ``
 - `` `[Events]` ``
 - `` `git --help` ``
 
-### File Names
+#### File Names
 
 When copying the name of a file, copy the letter casing as-is.
-For example:
+
+**Examples:**
 
 - `` `osu.exe` ``
 - `` `screenshot001.jpg` ``
 - `` `Opsu-android0.15.0b.apk` ``
 
-### File Extensions
+#### File Extensions
 
 When writing a file extension, prefix the extension with a period (`.`), followed by the file extension in lowercase letters.
 
+**Examples:**
+
 - `` `.jpg` ``
 - `` `.exe` ``
 - `` `.osz` ``
 
-### Chat Channels
+#### Chat Channels
 
 When copying the name of a chat channel, start it with a hash (`#`), followed by the channel name in lower case letters.
```

Comment:
They're slightly different though. Line 372 talks about chat channel names which are prefixed with a hash followed by their name in lowercase letters while line 700 talks about language names in how they must start with a capital letter.